workflow/rest/runs/runs.mdx

```diff
@@ -0,0 +1,389 @@
+---
+title: "List workflow runs"
+description: "Fetch details about workflow runs"
+api: "GET https://qstash.upstash.io/v2/workflows/events/runs"
+authMethod: "bearer"
+---
+
+You can fetch details about workflow runs, including their state, completed and in-progress steps, and step details.
+
+The retention duration for completed workflow runs depends on your quota. Please check the [pricing](https://upstash.com/pricing/qstash) page for details.
+
+<Note>
+If you are using custom workflow run IDs, this endpoint may return multiple runs when filtering by run ID.
+
+To uniquely identify a single workflow run, include the workflowCreatedAt timestamp in your filter.
+
+</Note>
+
+## Request
+
+<ParamField query="cursor" type="string">
+  By providing a cursor you can paginate through all of the workflow runs.
+</ParamField>
+
+<ParamField query="workflowRunId" type="string">
+  Filter workflow runs by run id.
+</ParamField>
+
+<ParamField query="workflowUrl" type="string">
+  Filter workflow runs by workflow url.
+</ParamField>
+
+<ParamField query="workflowCreatedAt" type="number">
+  Filter workflow runs by the unix milliseconds value of creation timestamp
+</ParamField>
+
+<ParamField query="state" type="string">
+  Filter workflow runs by [state](/qstash/howto/debug-logs)
```

Comment:
These are the states of the Workflow's right. Not the messages.
I guess we need to remove the link and mention that these can only be RUN_STARTED/RUN_SUCCESS etc.